.github/workflows/dsm-extract.yml

```
@@ -6,48 +6,63 @@ on:
         paths:
             - 'akeneo-design-system/**'
 jobs:
+
     build:
         name: Create a lib build
         runs-on: ubuntu-latest
         steps:
-            -   name: Checkout current branch
+            - name: Checkout the current branch
                 uses: actions/checkout@v2
-            -   name: Use Node.js 13
+
+            - name: Install node 13.13 for the rest of the workflow
                 uses: actions/setup-node@v1
                 with:
-                    node-version: 13.13
-            -   name: Install Packages
+                node-version: 13.13
+
+            -   name: Install node modules
                 run: cd akeneo-design-system && yarn install
+
             -   name: Build library
                 run: cd akeneo-design-system && yarn lib:build
-            -   uses: actions/upload-artifact@v1
+
+            -   name: Upload the generated lib to artifact
```

```suggestion
            -   name: Upload the generated lib as an artifact
```
Ou même "Github cloud" 😂